conf/viewvc.conf.dist

```diff
@@ -625,7 +638,7 @@
 ##   c      Context diff
 ##   s      Side by side
 ##   l      Long human readable (more context)
-##   f      Full human readable (entire file)
+##   f      Full human readable (very large context, entire file in most case)
 ##
```

Comment:
"...in most cases"

Reply:
Done in [6f7c0d2](https://github.com/viewvc/viewvc/pull/297/commits/6f7c0d2f1db140e49a50ef945eaca188f16bfbde)